src/lightning/pytorch/strategies/fsdp.py

```diff
@@ -267,7 +269,11 @@ def _setup_model(self, model: Module) -> Module:
         from torch.distributed.fsdp import FullyShardedDataParallel
 
         if any(isinstance(mod, FullyShardedDataParallel) for mod in model.modules()):
-            # the user wrapped at least one layer in `configure_model` already
+            # The user has wrapped their submodules manually, don't apply the auto wrap policy.
```

Comment:
This comment should go with the if below

Reply:
```suggestion
```